pyccel/codegen/printing/cwrappercode.py

```diff
@@ -400,7 +433,15 @@ def _print_PyClassDef(self, expr):
                 f"    .tp_methods = {method_def_name},\n"
                 "};\n")
 
-        return class_code + '\n' + function_sigs + '\n' + method_def + '\n' + type_code + '\n' + functions
+        return method_def + '\n' + type_code + '\n' + functions
+
+    def _print_PyModInitFunc(self, expr):
+        decs = ''.join(self._print(d) for d in expr.declarations)
+        body = self._print(expr.body)
+        return ''.join([f'PyMODINIT_FUNC {expr.name}(void)\n{{\n',
+                *decs,
+                *body,
+                '}\n'])
```

Comment:
I am confused by the unpacking of `decs` and `body`, which are strings. Can you please explain what is happening here?

Reply:
The result is the same as it splits it into characters and then joins then together again. It's just slower